src/main/io/display.c

```diff
@@ -99,6 +88,9 @@ static const char* const pageTitles[] = {
     "SENSORS",
     "RX",
     "PROFILE"
+#ifndef SKIP_TASK_STATISTICS
+    ,"TASKS"
```

Comment:
It's ok to have trailing comma in list. And its even better, only one line is affected when adding new item - much nicer in github diffs.